src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,89 @@
+package duke;
+
+import java.io.*;
```

Comment:
Perhaps you can import functions or classes explcitly in line with the CS2103/T coding style requirements instead of using a wildcard here?

Reply:
Have changed it